services/wfm/src/main/java/org/openkilda/wfm/topology/cache/service/CacheService.java

```diff
@@ -400,4 +402,17 @@ public void handleFlowEvent(FlowInfoData flowData, ISender sender, String correl
                 break;
         }
     }
+
+    /**
+     * Deletes a switch from a cache.
+     *
+     * @param request the instance of {@link SwitchDeleteRequest}.
+     * @param sender the instance of {@link Sender}.
+     * @param correlationId the log correlation id.
+     */
+    public void deleteSwitch(SwitchDeleteRequest request, Sender sender, String correlationId) {
```

Comment:
Not sure that adding the method 'deleteSwitch' is a good idea. Need to discuss with @Settis